tests/Oryx.BuildImage.Tests/NodeJSSampleAppsTest.cs

```diff
@@ -32,7 +32,7 @@ public void GeneratesScript_AndBuilds()
             var appDir = volume.ContainerDir;
             var appOutputDir = "/tmp/webfrontend-output";
             var script = new ShellScriptBuilder()
-                .AddBuildCommand($"{appDir} -o {appOutputDir}")
+                .AddBuildCommand($"{appDir} -i /tmp/int -o {appOutputDir}")
```

Comment:
These changes were made because `rsync` and directories which are volume mounted do not behave well together.